tools/lib/lolo_log_puller/lolo_log_puller_cli.py

```diff
@@ -97,31 +97,27 @@ def main():
         app_unit_clean_name = app_unit.replace('/', '-')
         # Note: /var/lib/charm may not always exist (ceph stores confs there)
 
-        cmd_archive = '''
-dpkg -l | bzip2 -9z > /home/ubuntu/{1}-dpkg-list.bz2 &&\
-
-[[ -d /var/lib/charm ]] && sudo tar -cjf /home/ubuntu/{1}-var-lib-charm.tar.bz2 /var/lib/charm ||: &&\
```

Comment:
Here you can see that there was `||:` so the original author was also confronted with the fact that `tar` sometimes exits non-null although it does the job. I have now added `||:` to all `tar` commands.